qemu/tests/cfg/usbdev.cfg

```diff
@@ -0,0 +1,149 @@
+    variants:
+        - without_usb_hub:
+        - with_usb_hub:
+            usb_devices += " hub1"
+            usb_type_hub1 = usb-hub
+            usb_port_hub1 = "2"
+            usb_port_testdev = "2.1"
+            usb_port_d0 = "2.1"
+            no usb-ehci
+            variants:
+                - one_usb_dev:
+                - max_usb_dev:
+                    no usb_hub
+                    no check
+                    usb_devices += " d1 d2 d3 d4 d5 d6 d7"
+                    udevs += " d1 d2 d3 d4 d5 d6 d7"
+                    usb_port_d1 = "2.2"
+                    usb_port_d2 = "2.3"
+                    usb_port_d3 = "2.4"
+                    usb_port_d4 = "2.5"
+                    usb_port_d5 = "2.6"
+                    usb_port_d6 = "2.7"
+                    usb_port_d7 = "2.8"
+
+    # usb devices
+    variants:
+        - usb_kbd:
+            usb_type = "usb-kbd"
+            info_usb_name = "QEMU USB Keyboard"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Keyboard"
+            usb_devices += " testdev"
+        - usb_mouse:
+            usb_type = "usb-mouse"
+            info_usb_name = "QEMU USB Mouse"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Mouse"
+            usb_devices += " testdev"
+        - usb_tablet:
+            usb_type = "usb-tablet"
+            info_usb_name = "QEMU USB Tablet"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Tablet"
+            usb_devices += " testdev"
+        - usb_ccid:
+            usb_type = "usb-ccid"
+            info_usb_name = "QEMU USB CCID"
+            vp_id = "08E6:4433"
+            vendor = "Gemplus"
+            product = "QEMU USB CCID"
+            usb_devices += " testdev"
+        - usb_hub:
+            usb_type = "usb-hub"
+            info_usb_name = "QEMU USB Hub"
+            vp_id = "0409:55aa"
+            vendor = ""
+            product = "QEMU USB Hub"
+            usb_devices += " testdev"
+        - usb_storage:
+            usb_type = "usb-storage"
+            info_usb_name = "QEMU USB MSD"
+            vp_id = "46f4:0001"
+            vendor = ""
+            product = "QEMU USB HARDDRIVE"
+            no piix3-uhci piix4-uhci ich9-uhci
+            max_usb_dev:
+                usb_devices = " tablet1 hub1"
+            udevs += " d0"
+            start_vm = no
+        - usb_bot:
+            usb_type = "usb-bot"    
+            info_usb_name = "QEMU USB MSD"
+            vp_id = "46f4:0001"
+            vendor = ""
+            product = "QEMU USB HARDDRIVE"
+            no piix3-uhci piix4-uhci ich9-uhci
+            max_usb_dev:
+                usb_devices = " tablet1 hub1"
+            udevs += " d0"
+            start_vm = no
+    # usb tests
+    variants:
+        - check:
+            type = usbdev_check
```

Comment:
Failed on ppc 

usbdev.check.usb_hub.without_usb_hub.ich9-ehci
05:52:39 ERROR|   File "/root/test_home/liuzt/source_code/staf-kvm-devel/workspace/avocado-vt/virttest/utils_test/qemu.py", line 1890, in check_usb_dev
05:52:39 ERROR|     raise exceptions.TestFail("Could not find item '%s' in guest" % item)
05:52:39 ERROR| TestFail: Could not find item 'QEMU USB Hub' in guest
05:52:39 ERROR|
05:52:39 ERROR| FAIL 18-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.usbdev.check.usb_hub.without_usb_hub.ich9-ehci -> TestFail: Could not find item 'QEMU USB Hub' in guest
Checked insides guest. The name of usb_hub is not existed indeed. So it is possible to add the exception?